File: shelter-frontend/src/app/shared/enums/stanowisko.enum.ts
export enum StanowiskoEnum {
  DYREKTOR = 'DYREKTOR',
  PRACOWNIK = 'PRACOWNIK',
  WOLONTARIUSZ = 'WOLONTARIUSZ',
  LEKARZ = 'LEKARZ',
}

export const StanowiskoLabels: Record<StanowiskoEnum, string> = {
  [StanowiskoEnum.DYREKTOR]: 'Dyrektor',
  [StanowiskoEnum.PRACOWNIK]: 'Pracownik',
  [StanowiskoEnum.WOLONTARIUSZ]: 'Wolontariusz',
  [StanowiskoEnum.LEKARZ]: 'Lekarz'
};

File: src/main/java/com/patneh/shelter/infrasructure/enums/StanowiskoEnum.java

package com.patneh.shelter.infrasructure.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
public enum StanowiskoEnum {
    DYREKTOR("Dyrektor"),
    PRACOWNIK("Pracownik"),
    WOLONTARIUSZ("Wolontariusz"),
    LEKARZ("Lekarz");

    @Getter
    private final String label;
}
